changelogs/unreleased/gh-6200-lua_cjson-encode-escapes-slash.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/lua/json
+
+* Added `json_escape_forward_slash` option to tarantool.compat and implemented
+  corresponding changes in lua-cjson and msgpuck (gh-6200).
```

Comment:
Our json built-in Lua module is based on lua-cjson, but was reworked. Anyway, it is an implementation detail. How about calling it '`json` built-in module' or '`json` Lua module' or just '`json`' in a text for a user?

Reply:
I still see it in the commit message title prefix (I would make it `lua/json:`), in the changelog entry file name, in the test file name. Not a big problem, but I would polish it for accuracy and consistency.

BTW, json is Lua built-in module, but msgpuck is not:

> in json and msgpuck built-in modules

I would say 'json built-in module and msgpuck library' or, better (from a user perspective) 'json built-in module and JSON logger type'.